test/SourceKit/DocSupport/Inputs/cake.swift

```diff
@@ -79,4 +79,7 @@ public protocol P4 {}
 
 extension C1 : P4 {
   public func C1foo() {}
+  public struct C1S1{
```

Comment:
Can you add tests for generic nested types and extensions of generic types also?